opensrp-path/src/main/java/org/smartregister/path/provider/ChildSmartClientsProvider.java

```diff
@@ -302,37 +302,37 @@ private void updateRecordVaccination(VaccineViewRecordUpdateWrapper updateWrappe
             recordVaccination.setEnabled(false);
 
         } else if (state.equals(State.WAITING)) {
-            recordVaccinationText.setText("Waiting");
+            recordVaccinationText.setText(R.string.waiting_label);
             recordVaccinationText.setTextColor(context.getResources().getColor(R.color.client_list_grey));
 
             recordVaccination.setBackgroundColor(context.getResources().getColor(R.color.white));
             recordVaccination.setEnabled(false);
         } else if (state.equals(State.EXPIRED)) {
-            recordVaccinationText.setText("Expired");
+            recordVaccinationText.setText(R.string.expired_label);
             recordVaccinationText.setTextColor(context.getResources().getColor(R.color.client_list_grey));
 
             recordVaccination.setBackgroundColor(context.getResources().getColor(R.color.white));
             recordVaccination.setEnabled(false);
         } else if (state.equals(State.UPCOMING)) {
-            recordVaccinationText.setText("Due\n" + stateKey);
+            recordVaccinationText.setText(context.getString(R.string.upcoming_label) + System.getProperty("line.separator") + stateKey);
```

Comment:
actually, is there a constant folder we can put a `LINE_SEPARATOR`? instead of fetching this property more than once, which should be constant for the run-time of the app

Reply:
Yea, there is. Just made the appropriate changes. Thanks